providers/default.rb

```diff
@@ -434,15 +434,38 @@ def open_handle_functions
 def event_log_ps_code
   powershell_script 'EventLog Restart' do
     code <<-EOH
-      $windows_kernel_version = (Get-WmiObject -class Win32_OperatingSystem).Version
-      if (-Not ($windows_kernel_version.Contains('6.0') -or $windows_kernel_version.Contains('6.1'))) {
-        $dependentActiveServices = (Get-Service EventLog).DependentServices | Where-Object {$_.Status -eq "Running"}
-        Stop-Service -Name "EventLog" -Force
-        Start-Service -Name $dependentActiveServices.Name
-        if((Get-Service EventLog).Status -ne 'Stopped' ){
-          Start-Service -Name "EventLog"
+    $windows_kernel_version = (Get-WmiObject -class Win32_OperatingSystem).Version
```

Comment:
This is fine but could be improved as `(Get-CimInstance -classname Win32_OperatingSystem).Version`.